docs/howto/error-reporting.txt

```diff
@@ -305,6 +305,63 @@ following attributes and methods:
         traceback frame. Sensitive values are replaced with
         :attr:`cleansed_substitute`.
 
+If you need to customize error reports beyond filtering you may specify a
```

Comment:
Maybe `versionadded` before this section :thinking:

Reply:
This is a good idea. It adds a good "New in ..." header.